### ISSUE TITLE
flint-2.8.0-beta1 testsuite failure on cygwin

### ISSUE THREAD

**Issue by mkoeppe** [wbhart/flint2#974]
```
set_fmpz_mat....PASS
mkdir -p build/interfaces/test
if [ "1" -eq "1" ]; then \
	make build/interfaces/test/t-NTL-interface; \
	build/interfaces/test/t-NTL-interface; \
fi
make[4]: *** No rule to make target 'build/interfaces/test/t-NTL-interface'.
/bin/sh: line 2: build/interfaces/test/t-NTL-interface: No such file or directory
```
https://github.com/mkoeppe/flint2/runs/3037336754

**Comment by wbhart** [comment]
Hi Professor Köppe,

I will probably need some more information here. The rule to make this target is on line 250 of Makefile.in, but as you can see, it has $(EXEEXT) appended. 

Could you look in your Makefile and see what EXEEXT is on line 15. It looks like it is expected to be nothing, but presumably it is something on the Cygwin you are using. It's not clear that EXEEXT is actually used everywhere it should be, which would explain this.

Perhaps you could confirm if adding $(EXEEXT) at the end of the filename in line 194 of Makefile fixes the problem for you.

**Comment by wbhart** [comment]
Fixed in trunk and 2.8.0-rc2.

**Comment by mkoeppe** [comment]
Great, thanks a lot!
